test/UnitTest.cs

```diff
@@ -2256,73 +2256,75 @@ public void Gather()
         [TestMethod]
         public void Redistribute()
         {
+            var option = new QueryOption();
+            option.optimize_.enable_broadcast_ = false;
```

Comment:
how about adding a loop on turn on/off enable_broadcast_ and assert either way the results are the same. If the plan is different, assert the difference. See similar usage in tpch tests.

Reply:
sure, can I create a broadcast test method to do that?